sktime/forecasting/compose/_reduce.py

```diff
@@ -895,15 +895,25 @@ def _predict_last_window(
             fh_max = fh.to_relative(self.cutoff)[-1]
 
             y_pred = np.zeros(fh_max)
+
+            # Array with input data for prediction.
             last = np.zeros((1, n_columns, window_length + fh_max))
 
             # Fill pre-allocated arrays with available data.
             last[:, 0, :window_length] = y_last
             if X is not None:
-                last[:, 1:, :window_length] = X_last.T
-                last[:, 1:, window_length:] = X.iloc[
-                    -(last.shape[2] - window_length) :, :
-                ].T
+                X_to_use = np.concatenate(
+                    [X_last.T, X.iloc[-(last.shape[2] - window_length) :, :].T], axis=1
```

Comment:
is this fine for a positive and a negative number and zero too?

Reply:
`last.shape` is always greater than window_length.  (see line 900). Thus, `(last.shape[2] - window_length)` is always positive.

Reply:
ok, thx